pkg/config/utils.go

```diff
@@ -51,7 +52,7 @@ func FindAllStackConfigsInPathsForStack(
 		if len(allMatches) == 0 {
 			_, err := u.GetGlobMatches(patterns[0])
 			if err != nil {
-				if atmosConfig.Logs.Level == u.LogLevelTrace {
+				if atmosConfig.Logs.Level == u.LogLevelDebug {
```

Comment:
Why are we using log levels outside of logging?

Reply:
We are litterally changing the return behavior depending on the log level, and I assume no tests.